Dashboard/Dockerfile

```diff
@@ -7,14 +7,23 @@ RUN mkdir .streamlit
 COPY .streamlit/config.toml .streamlit
 
 RUN mkdir logo
-COPY logo/hermes_crop.png logo
-COPY logo/hermes_logo_full.png logo
+COPY logo/asos_logo.png logo
+COPY logo/asos_logo.svg logo
+COPY logo/hermes_icon_crop.png logo
+COPY logo/hermes_icon.png logo
+COPY logo/hermes_logo_black.png logo
+COPY logo/hermes_logo_orange.png logo
 COPY logo/hermes_logo.png logo
+COPY logo/patagonia_bear.png logo
+COPY logo/patagonia_logo.png logo
```

Comment:
Remove the images that are not needed

Reply:
I'm going to create a new pull request for this! Thank you for notifying me!